src/test/java/componenttest/FeatureTest.java

```diff
@@ -0,0 +1,21 @@
+package componenttest;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+
+public class FeatureTest extends AbstractComponentTest {
+
+    // Just an example
+    @Test
+    public void featureref_returns_some_features() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2221564903/issues/source?bid=3131965&fileBranchId=3173282#l15)